test/framework/envs/universal/env.go

```diff
@@ -67,3 +68,12 @@ func PrintCPLogsOnFailure(report ginkgo.Report) {
 		}
 	}
 }
+
+func ExceptCpToNotPanic() {
+	logs, err := Cluster.GetKumaCPLogs()
+	if err != nil {
+		framework.Logf("could not retrieve cp logs")
+	} else {
+		Expect(strings.Contains(logs, "runtime.gopanic")).To(BeFalse())
```

Comment:
I don't know if that catches all panics. This one happened in the goroutine, what if there is other panic? Example issue from the past: https://github.com/kumahq/kuma/issues/5552

```
panic: expected struct or pointer to a struct got ptr                                                                                                                                                   
                                                                                                                                                                                                       
goroutine 1663 [running]:   
```

Reply:
I was afraid of just using "panic" because it might catch "lb_healthy_panic_threshold` or other Envoy strings with "panic" - changed that to include also "panic:"